packages/inquirer-common/src/prompts/cf-helper.ts

```diff
@@ -26,7 +26,14 @@ export async function getCFAbapInstanceChoices(
             AbapEnvType.ABAP_STAGING,
             AbapEnvType.ABAP_INTERNAL_STAGING
         ];
-        const serviceInstanceInfo: ServiceInstanceInfo[] = await getServicesInstances(filteredInstances);
+        // Load additional ABAP service types to extend the filtered instance list
+        const envFilteredInstances = process.env.ABAPEnvServiceTypes
```

Comment:
Can we pass as an option instead of loading from the env var? The consuming generator should load and pass.

Reply:
Based on further investigation, the existing filtered list is not static and can change from different subaccount. For now, the approach to use an env variable to update the list will be required and a TBI will be generated to investigated how best to handle this without requiring any change from the user.